core/trino-main/src/test/java/io/trino/operator/aggregation/BenchmarkDecimalAggregation.java

```diff
@@ -39,6 +39,7 @@
 import org.testng.annotations.Test;
```

Comment:
this should be separate PR

Reply:
all changes to `BenchmarkDecimalAggregation`?

Reply:
@sopel39 ping

Reply:
extracted `BenchmarkDecimalAggregation` changes to https://github.com/trinodb/trino/pull/13939